jvb/src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java

```diff
@@ -160,13 +160,13 @@ boolean updateLastNEndpoints()
     {
         synchronized (syncRoot)
         {
-            Map<Boolean, List<AbstractEndpoint>> bySendingVideo
+            Map<Boolean, List<AbstractEndpoint>> byVideoAvailable
                     = endpointsBySpeechActivity.stream()
-                        .collect(Collectors.groupingBy(AbstractEndpoint::isSendingVideo));
+                        .collect(Collectors.groupingBy(ep -> ep.getVideoType() != VideoType.NONE));
```

Comment:
Kotlin has a nice [`partition`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/partition.html) function that splits a collection into 2 based on a predicate which would be nice here.  Maybe someday when `ConferenceSpeechActivity` gets ported.